src/lib/apparmor/profiles.rb

```diff
@@ -248,6 +248,14 @@ def table
       )
     end
 
+    # Returns the list of visible profiles
+    #
+    # Each element of the array is another array with the profile name
+    # and a profile object.
+    #
+    # [["apache2", #<AppArmor::Profile...>], ["dovecot", #<AppArmor::Profile...>]]
```

Comment:
Thanks for the clarification, it helps to understand the above `first&.last` :wink: